powdr_cli/src/main.rs

```diff
@@ -779,7 +779,7 @@ mod test {
         {
             let file = output_dir
                 .path()
-                .join("simple_sum_opt.pil")
+                .join("simple_sum.pil")
```

Comment:
Otherwise, the command would look for `simple_sum_opt_constants.bin`...
The Pipeline always optimizes PIL anyway before doing anything with it.